models/sudoswap/ethereum/sudoswap_ethereum_events.sql

```diff
@@ -415,6 +415,6 @@ SELECT
     , royalty_fee_percentage
     , royalty_fee_currency_symbol
     , royalty_fee_receive_address
-    , 'sudoswap-' || tx_hash || '-' || nft_contract_address || raw_token_id::string || '-' || seller || '-' || amount_original::string || 'Trade' AS unique_trade_id
+    , 'sudoswap-' || tx_hash || '-' || nft_contract_address || token_id::string || '-' || seller || '-' || amount_original::string || 'Trade' AS unique_trade_id
```

Comment:
side note: while we're in this script, can we replace any instances of `::` for casting datatypes to use `cast()` to continue preparing for duneSQL?